README.md

```
@@ -124,9 +124,9 @@ Meet our wonderful team comprised of Product Managers, Designers, Tech Leads, En
             </td>
             <td align="center" width="150">
                 <a href="https://umd.hack4impact.org/">
-                <img src="public/team-photos/pending-image.jpg" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
-                <b>Anna</b><br/><br/>
-                <img src="https://img.shields.io/badge/💻_engineer-27AE60?style=flat-square" alt="Engineer"/>
+                <img src="public/team-photos/AnnaLin.png" alt= "Anna Lin's Team Headshot" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
+                <b>Anna Lin</b><br/><br/>
+                <img src="https://img.shields.io/badge/💻_engineer-27AE60?style=flat-square"/>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)